Sources/JWTKeychain/Support/UserAuthenticating.swift

```diff
@@ -50,7 +56,7 @@ public class UserAuthenticator: UserAuthenticating {
     /// - Returns: the updated user
     public func update(request: Request, hasher: HashProtocol) throws -> U {
```

Comment:
I know it's not part of your PR, but if you're up for changing it, I think we should use camelCase keys - ok with you?